packages/react/src/utilities/decorators/withViewport.tsx

```diff
@@ -52,6 +52,13 @@ export interface IWithViewportProps {
    * This has no impact if `skipViewportMeasures` is `true`, as no viewport measurement strategy is used.
    */
   disableResizeObserver?: boolean;
+  /**
+   * Whether or not `withViewport` will delay before first measuring the viewport size.
+   * Setting this will delay measurement by the same amount as the debounce for resizing the window.
+   * This is useful for giving the child of the viewport time to render before measuring.
+   * @default false
+   */
```

Comment:
Might be worth it to call out what are the type of scenarios that require us to default it to false.